src/helper.cc

```diff
@@ -1207,8 +1207,10 @@ Enqueue(helper * hlp, Helper::Xaction * r)
 
     /* do this first so idle=N has a chance to grow the child pool before it hits critical. */
     if (hlp->childs.needNew() > 0) {
-        debugs(84, DBG_CRITICAL, "Starting new " << hlp->id_name << " helpers...");
-        helperOpenServers(hlp);
+        CallService(nullptr, [&hlp] {
+            debugs(84, DBG_CRITICAL, "Starting new " << hlp->id_name << " helpers...");
```

Comment:
It feels odd that we place this "Starting new..." message inside the service context but leave the "WARNING: All ... processes are busy" message below it in the current transaction context. Both messages seem to have the same "transaction-agnostic" or "about many transactions" qualities. Maybe we should surround the WARNING-producing code below with the same CallService() wrapper?